src/module/so-inspired.js

```diff
@@ -23,6 +23,75 @@ Hooks.on("updateUser", (user) => {
   updateSheetForInspo(user);
   //updatePlayerListInspo();
 });
+
+Hooks.on("tidy5e-sheet.renderActorSheet", (app, element, data) => {
+  if (app.actor.type !== "character") {
+    return;
+  }
+
+  const html = $(element);
+
+  const inspirationArea = html.find(".inspiration");
+  inspirationArea.hide();
+
+  let currentInspiration;
+  const actorOwner = game.users.find(
```

Comment:
For this PR, I just copied a lot of the functionality from the main logic. Please feel free to extract the common behaviors into functions if that works better for you.